sonar-project.properties

```diff
@@ -13,4 +13,4 @@ sonar.typescript.tsconfigPath=.
 # Comma-delimited list of paths to LCOV coverage report files. Paths may be absolute or relative to the project root.
 sonar.javascript.lcov.reportPaths=./coverage/lcov.info
 
-sonar.exclusions=**/*.test.ts
+sonar.exclusions=**/*.test.ts, **/*.test.tsx
```

Comment:
We may actually want to use `sonar.coverage.exclusions` if the goal is to just exclude from the code coverage metric

Reply:
I believe that putting code in the test scope prevents it being included in coverage. I think the `sonar.coverage.exclusions` must be to additionally exclude files from the sources scope which would otherwise be included. Looking at https://docs.sonarqube.org/latest/project-administration/narrowing-the-focus/ I think the solution is:
```
sonar.sources=src
sonar.exclusions=**/*.test.ts, **/*.test.tsx
sonar.tests=src
sonar.test.inclusions=**/*.test.ts, **/*.test.tsx
```
Does that make sense?